tests/unit/contracts/test_independence.py

```diff
@@ -59,7 +69,12 @@ def test_when_modules_are_independent(self):
         )
 
         contract_check = self._check_default_contract(graph)
+        assert contract_check.kept
+
+    def test_when_wildcard_modules_are_independent(self):
```

Comment:
We could do with a second test to assert that `contract_check.kept is False`, given a graph that should trigger a failure.